NotificationSystem/SendModeratorEmail.cs

```diff
@@ -7,7 +7,10 @@
 using Microsoft.Extensions.Logging;
 using NotificationSystem.Template;
 using NotificationSystem.Utilities;
-using SendGrid.Helpers.Mail;
+using Amazon;
+using Amazon.SimpleEmail;
+using RateLimiter;
+using ComposableAsync;
```

Comment:
Please move these up so the using statements are all in alphabetical order.

Reply:
Moved up